snap/snapcraft.yaml

```diff
@@ -178,14 +178,8 @@ parts:
         TARGET=kata-ksm-throttler
 
   kernel:
-    override-pull: |
-      versions_url=https://raw.githubusercontent.com/kata-containers/runtime/${SNAPCRAFT_PROJECT_VERSION}/versions.yaml
-      version="$(curl -sSL ${versions_url} | yq r - assets.kernel.version | tr -d v)"
-      url="$(curl -sSL ${versions_url} | yq r - assets.kernel.url)"
-      curl -LO ${url}/linux-${version}.tar.xz
-      tar -xf linux-${version}.tar.xz --strip-components=1
-    after: [kernel-dump]
-    plugin: kernel
+    after: [kernel-dump, scripts-dump]
+    plugin: nil
```

Comment:
I use `nil` to avoid an unexpected behaviour, I wouldn't like to see `snapcraft` using a default plugin to build a package that doesn't need it. `nil` means _don't pull the source code, I'll pull and build it_

Reply:
Thanks for explaining @devimc.